### ISSUE TITLE
Cannot pull in other efficientnet versions

### ISSUE THREAD

**Issue by craine** [walkwithfastai/walkwithfastai.github.io#12]
Hi. 
I'm pulling in:
from wwf.vision.timm import *

When I ran your example:
timm.list_models('*efficientnet*')[:10]

I see various models but for some reason even though I see b4 when I try to run it I get:
Pretrained model URL is invalid, using random initialization.
<fastai.learner.Learner at 0x7fe259fdd9b0>

But I don't get that with any b3 models. 

This is my learn:
learn = timm_learner(dls, 'efficientnet_b4', loss_func=LabelSmoothingCrossEntropy(), metrics=[accuracy], opt_func=ranger)

**Comment by muellerzr** [comment]
Which URL are you trying for b4? The only pretrained weights for it are the
TF weights. (That’s a timm thing)

**Comment by craine** [comment]
I wasn't using any specific URL.. I just did the view timm models and saw efficientnet_b4 and thought I could pull it in like that.  I guess I'm not clear from your tutorial:
https://walkwithfastai.com/vision.external.timm

It shows searching for models. All those models aren't available when using your from wwf.vision.timm import * ?

**Comment by muellerzr** [comment]
My timm module interacts with his library. What you searched for was a list
of *architectures*, not pretrained models. So while efficientnet_b4 is a
valid architecture, it doesn’t have normal pretrained weights that he
trained. To use pretrained b4 weights you need to use the TF ported
weights.

**Comment by craine** [comment]
Ah! Got it. 
I see this:
body = create_timm_body('efficientnet_b3a', pretrained=True)

How do I see what is available that are pre-trained models?

**Comment by muellerzr** [comment]
I would recommend reading through Ross' library here instead, as WWF doesn't directly do anything special. It just uses timm to generate the model through its own system (then wraps and adjusts the head to be fastai): https://github.com/rwightman/pytorch-image-models

**Comment by muellerzr** [comment]
The same string calls are being used.

**Comment by muellerzr** [comment]
For exact configurations you can look in the exact model's definition file, such as here for effnet's: https://github.com/rwightman/pytorch-image-models/blob/master/timm/models/efficientnet.py
